heidegger_index/views.py

```diff
@@ -90,7 +95,21 @@ def get_context_data(self, **kwargs):
         lemma = context["lemma"]
         context["children"] = lemma.children.all()
         context["related"] = lemma.related.all()
+        try:
+            lemma_urn = CTS_URN(lemma.urn)
+            if lemma.type == "w" and lemma_urn.passage_component:
+                p_link = 'https://scaife-cts.perseus.org/api/cts?request=GetPassage&urn=' + lemma.urn
+                p_response = requests.get(p_link)
```

Comment:
Jammer hieraan is dat het laden van de lemma pagina wacht op een externe API call. Ik denk dat het mooier is om dit te prepopulaten in de database (net als met de citeproc references).

Reply:
Deze staat nu in [`populate_index.py`](https://github.com/andredelft/heidegger-index/pull/20/files#diff-e5011d5d7afce6b37513d7f58b4587de62679cb4ea782dcf1779e6f02681015e)